modules/src/xibo-finance-render.js

```diff
@@ -33,6 +33,8 @@ jQuery.fn.extend({
       previewHeight: 0,
       scaleOverride: 0,
     };
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
URLSearchParams is not available in older versions of webOS/Tizen.

We this polyfilled in xiboIC, so we should use that to determine whether we're the editor or not?

Reply:
The thing is, `xiboIC.checkIsPreview()` will always return **true** even when we are in the editor since it first evaluates the `preview` param.

```
checkIsPreview: function() {
      // If we don't have URLSearchParams defined, we're also in preview
      if (typeof(URLSearchParams) === 'undefined') {
        _lib.isPreview = true;
        return true;
      }

      // Check if we have the preview flag in URL
      const searchParams = new URLSearchParams(window.location.search);
      if (searchParams.has('preview') && searchParams.get('preview') == 1) {
        _lib.isPreview = true;
        return true;
      }

      // For the widget preview in viewer
      if (searchParams.has('isEditor') &&
        searchParams.get('isEditor') === '1'
      ) {
        _lib.isPreview = true;
        return true;
      }

      return false;
    }
```

Reply:
Should we update `xiboIC.checkIsPreview()` to first evaluate for `isEditor` param?

Reply:
This makes more sense than updating the xiboIC.checkIsPreview(). Let me add it to xiboIC then